.env.example

```diff
@@ -13,4 +13,7 @@ DB_NAME=
 SMTP_HOST=
 SMTP_PORT=
 SMTP_USERNAME=
-SMTP_PASSWORD=
\ No newline at end of file
+SMTP_PASSWORD=
+
+# TEMP
+CREATED_BY=
```

Comment:
Can make it APP_NAME?

And add the comment saying to give the name of the app for which this is being used.

If pinestem then PINESTEM  will be value for this.

Reply:
ok

Reply:
Updated